exercises/practice/yacht/test_yacht.c

```diff
@@ -0,0 +1,250 @@
+#include "yacht.h"
+
+#include "test-framework/unity.h"
+
+void setUp(void)
+{
+}
+
+void tearDown(void)
+{
+}
+
+static void test_yacht(void)
+{
+   const dice_t dice = { 5, 5, 5, 5, 5 };
+   TEST_ASSERT(score(dice, YACHT) == 50);
+}
+
+static void test_not_yacht(void)
+{
+   TEST_IGNORE();
```

Comment:
In most exercises there's a comment in this first `TEST_IGNORE()` line that explains that this should be deleted to enable the tests, e.g. in [test_difference_of_squares.c](https://github.com/exercism/c/blob/main/exercises/practice/difference-of-squares/test_difference_of_squares.c#L19)
Do you mind adding such comment?

Reply:
Done!